### ISSUE TITLE
Unable to build executable

### ISSUE THREAD

**Issue by sgourdas** [kiwix/kiwix-desktop#1124]
When doing `sudo make install` I get the below:

```
src/localkiwixserver.cpp: In constructor ‘LocalKiwixServer::LocalKiwixServer(QWidget*)’:
src/localkiwixserver.cpp:44:66: error: ‘const class std::__cxx11::basic_string<char>’ has no member named ‘addr’
   44 |         QString ip = QString::fromStdString(interfacePair.second.addr);
      |                                                                  ^~~~
src/localkiwixserver.cpp: In member function ‘void LocalKiwixServer::runOrStopServer()’:
src/localkiwixserver.cpp:91:117: error: too many arguments to function ‘std::string kiwix::getBestPublicIp()’
   91 | ") ? m_ipAddress : QString::fromStdString(kiwix::getBestPublicIp(false));
      |                                                                       ^

In file included from src/localkiwixserver.cpp:4:
/usr/local/include/kiwix/tools.h:221:13: note: declared here
  221 | std::string getBestPublicIp();
      |             ^~~~~~~~~~~~~~~
```

Seems to be coming from #1122

**Comment by veloman-yunkan** [comment]
If you build `libkiwix` on your own, just update it. Otherwise you will have to wait until `kiwix/libkiwix#1093` is merged whereupon #1122 may be reverted.

**Comment by sgourdas** [comment]
Got it. Thank you.
